src/Durable/DurableBindings.cs

```diff
@@ -13,9 +13,13 @@ internal static class DurableBindings
         private const string OrchestrationTrigger = "orchestrationTrigger";
         private const string ActivityTrigger = "activityTrigger";
 
+        // For Durable v1 only
+        private const string OrchestrationClient = "orchestrationClient";
+
         public static bool IsDurableClient(string bindingType)
         {
-            return string.Compare(bindingType, DurableClient, StringComparison.OrdinalIgnoreCase) == 0;
+            return string.Compare(bindingType, DurableClient, StringComparison.OrdinalIgnoreCase) == 0
+                   || string.Compare(bindingType, OrchestrationClient, StringComparison.OrdinalIgnoreCase) == 0;
```

Comment:
Do we log anywhere what Durable version is the customer using? If this information could be useful for debugging (or a CRI investigation), please add it. Otherwise, let's keep it as is.

Reply:
I agree this would be useful, but let's do it separately: #446 filed